test/FileService.Business.UnitTests/AddNewImageCommandTests.cs

```diff
@@ -108,48 +113,55 @@ public void SetUp()
         }
 
         [Test]
-        public void ShouldAddNewImageAndThumbImmage()
+        public void ShouldAddThumbImageWhenImageIsBig()
         {
+            isBigImage = true;
```

Comment:
remove

Reply:
this is local condition for this test. without it test doesn't work, we did this on scrum on monday